src/libs/API/types.ts

```diff
@@ -679,6 +679,7 @@ const READ_COMMANDS = {
     OPEN_POLICY_INITIAL_PAGE: 'OpenPolicyInitialPage',
     SEARCH: 'Search',
     OPEN_SUBSCRIPTION_PAGE: 'OpenSubscriptionPage',
+    EXPORT_SEARCH_ITEMS_TO_CSV: 'ExportSearchToCSV',
```

Comment:
Let's move this to `WRITE_COMMANDS`, because when exporting a report we will create a `reportAction`.

Reply:
Done